apps/auth_content/admin.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib import admin
+
+from import_export.admin import ExportMixin
+
+from .models import ClosedPODContactInformation
+
+
+@admin.register(ClosedPODContactInformation)
+class ClosedPODContactInformationAdmin(ExportMixin, admin.ModelAdmin):
+    list_display = (
+        "id",
+        "user",
+        "facility_name",
+        "facility_id",
+        "primary_contact_name",
```

Comment:
Could we change the way the `primary_contact_name` and `secondary_contact_name` are displayed? Currently it's just "NAME*" and "NAME", which doesn't really explain what they are.
Or maybe it's easier to just not show them?

Reply:
done